processing/src/test/java/io/druid/query/aggregation/AggregationTestHelper.java

```diff
@@ -387,12 +390,12 @@ public void createIndex(
 
   public Sequence<Row> runQueryOnSegments(final List<File> segmentDirs, final Query query)
   {
-    final List<Segment> segments = Lists.transform(
+    final List<AbstractSegment> segments = Lists.transform(
```

Comment:
Why are these changes required?  I think that keeping it at just `Segment` should be fine (and more correct, 'cause it's totally possible that someone chooses to implement `Segment` without using `AbstractSegment`).